v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/transforms/AvroTransform.java

```diff
@@ -61,15 +66,27 @@
  */
 public class AvroTransform
     extends PTransform<PCollection<KafkaRecord<byte[], byte[]>>, WriteResult> {
-
-  private static final Logger LOG = LoggerFactory.getLogger(AvroTransform.class);
-
+  private BadRecordRouter badRecordRouter = BadRecordRouter.RECORDING_ROUTER;
+  private List<ErrorHandler<BadRecord, ?>> errorHandlers;
```

Comment:
Code style nit: don't mix class variable and static variables, i.e. badRecordHandler, errorHandlers, and options should be declared together at the bottom.

Reply:
Also make all of them final if possible.